docs/docs/features/dongle.mdx

```diff
@@ -0,0 +1,349 @@
+- Added latency due to the extra hop between the host device and the keyboard halves compared to a USB connected non-dongle setup.
+
+
+
+## Defining a Dongle for a Keyboard 
+
+
+:::note
+In the examples below, we will be referencing the New Keyboard Shield guide files and will be building on top of that.
+Both split and unibody keyboards can be used with a dongle.
+:::
+
+### Keyboard Shield
+
+<Tabs
+defaultValue="split"
+values={[
+{label: 'Split Keyboard', value: 'split'},
+{label: 'Unibody Keyboard', value: 'unibody'},
+]}
+>
+
+<TabItem value="split">
+
+First we will introduce a third split to our keyboard configuration. This will be used as the dongle.
+
+```kconfig title="Kconfig.shield"
+config SHIELD_MY_BOARD_DONGLE
+    def_bool $(shields_list_contains,my_board_dongle)
+
+config SHIELD_MY_BOARD_LEFT
+    def_bool $(shields_list_contains,my_board_left)
+
+config SHIELD_MY_BOARD_RIGHT
+    def_bool $(shields_list_contains,my_board_right)
+```
+</TabItem>
+<TabItem value="unibody">
+We will add a dongle to the unibody keyboard configuration. 
+
+```kconfig title="Kconfig.shield"
+config SHIELD_MY_BOARD_DONGLE
+    def_bool $(shields_list_contains,my_board_dongle)
+
+config SHIELD_MY_BOARD
+    def_bool $(shields_list_contains,my_board)
+
+```
+</TabItem>
+</Tabs>
+
+### Keyboard Defconfig
+
+:::info
+In the shield defconfig we will assume that the user does not want to use the dongle by default.
+This is done so that the user can easily enable the dongle in the user configuration without having to modify the shield configuration.
+:::
+
+<Tabs
+defaultValue="split"
+values={[
+{label: 'Split Keyboard', value: 'split'},
+{label: 'Unibody Keyboard', value: 'unibody'},
+]}
+>
+
+<TabItem value="split">
+
+Next we will define the roles of the keyboard halves. The left and right halves will be set as peripherals, and the dongle will be set as the central.
+We will also only give the central device the keyboard name.
+
+There is also an option to still use the left half as the central, and the right half as the peripheral. In that case the dongle will not be used.
+
+
+```kconfig title="Kconfig.defconfig"
+if SHIELD_MY_BOARD_DONGLE
+
+    config ZMK_KEYBOARD_NAME
+        default "My Board"
+
+    config ZMK_SPLIT_ROLE_CENTRAL
+        default y
+
+    config ZMK_SPLIT_BLE_CENTRAL_PERIPHERALS
+        default 2
+
+    # Prevent the dongle from going to sleep
+    config ZMK_SLEEP
+        default n
+
+    # Increase the transmit power of the dongle
+    config BT_CLTR_TX_PWR_PLUS_8
+        default y
+
+endif
+
+
+if SHIELD_MY_BOARD_LEFT
+
+    # Setting this to y will make the left half the central
+    # We want it to be the central when dongle is not used
+    config ZMK_SPLIT_ROLE_CENTRAL
+        default y
+
+    # Check if the left half is the central
+    if ZMK_SPLIT_ROLE_CENTRAL
+        config ZMK_KEYBOARD_NAME
+            default "My Board"
+            
+    endif # ZMK_SPLIT_ROLE_CENTRAL
```

Comment:
Thinking on this some more, I think it would be better if we allowed the left/unibody to keep their name and rename the dongle to "My ZMK Dongle". That would simplify this.

Reply:
I think I prefer the current solution more, it feels more finished compared to "My Board Dongle", seems like it is part of the package